Broken npmjs itowns package 2.45.0

<!--- Provide a general summary of the issue in the Title above -->
Hello,
I recently updated itowns in our custom 3D viewer, but it complained about missing files inside the lib folder specifically.
After checking on npmjs, I saw that the last packages since the 2.45.0 version are missing every file except for examples. ([https://registry.npmjs.com/itowns/-/itowns-2.45.0.tgz](https://registry.npmjs.com/itowns/-/itowns-2.45.0.tgz))

Did something change in the way we need to import the new packages ?

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: 2.45.0
* Browser Name and version: Not relevant
* Operating System and version (desktop or mobile): Fedora 41
* Link to your project: Not publicly accessible

## Context
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
Simply updating the package prevents us from using it.
<!--- How has this bug affected you? -->
I couldn't use the last itowns version.
<!--- If you have errors or a stacktrace in the console, here is a good place to put it. -->
```
[ERROR] Failed to resolve entry for package "itowns". The package may have incorrect main/module/exports specified in its package.json.
```
<!--- Please consider using a pastebin service or create a gist.-->
<!--- Some screenshots are always greatly appreciated here! -->


## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. Download the 2.45.0 npmjs package (https://registry.npmjs.com/itowns/-/itowns-2.45.0.tgz)

## Expected Behavior
<!--- Tell us what you think should happen -->
Have a package ready to use with every file it needs.

## Actual Behavior
<!--- Tell us what happens instead of the expected behavior -->
Nothing.

## Possible Cause/Fix/Solution
<!--- Not obligatory, but suggest a cause or reason for the bug if relevant, -->
<!--- or ideas how to implement the addition or change -->
<!--- In any case, please consider opening a PR and become an iTowns developer! -->

--- a/package.json
+++ b/package.json
@@ -31,8 +31,8 @@
     "changelog": "conventional-changelog -n ./config/conventionalChangelog/config.cjs -i changelog.md -s",
     "bump": "if [ -z $npm_config_level ]; then grunt bump:minor;  else grunt bump:$npm_config_level; fi && npm run changelog && npm run update-packages && npm install && git add -A && git commit --amend --no-edit",
     "prerelease-next": "npm version prerelease --preid next && npm run prerelease-next --workspaces --if-present && npm run update-packages",
-    "publish-next": "npm run prerelease-next && npm run publish-next --workspaces --if-present",
-    "publish-latest": "npm run publish-latest --workspaces --if-present",
+    "publish-next": "npm run prerelease-next && npm run prepublishOnly && npm run publish-next --workspaces --if-present",
+    "publish-latest": "npm run prepublishOnly && npm run publish-latest --workspaces --if-present",
     "update-packages": "npm run update-package --workspaces --if-present"
   },
   "c8": {

